### PR TITLE
[Feature] #30 Add function that returns safeArea top value

### LINKED ISSUE #30: [Feature] Add code that returns safeArea top value
# safeArea Issue
   - The code that returns safeArea top value in SafeAreaPadding folder

## 🤔 Completion conditions

- [ ] returns top of the safe area to suit the device

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number
🔒 Close #30 

## Changes

- Changed files
    - campair/campair/Application/SafeAreaPadding.swift
    - campair/campair/View/Editor/EditorDetailView.swift

## New features

- Add function that returns safeArea top value

## Review points

- I made a function to get the safeArea top value. Please check the link below for how to use it.
https://github.com/DeveloperAcademy-POSTECH/MC2-Team2-GreedySquirrel/wiki/06.14.tue-:-safeArea-Issue

## References

- https://www.codegrepper.com/code-examples/swift/swiftui+detect+notch
- https://stackoverflow.com/questions/52402477/ios-detect-if-the-device-is-iphone-x-family-frameless

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by sustainable-git — APPROVED** [review]
GGOODD!

File: campair/campair/View/Editor/EditorDetailView.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ struct OpeningEditorView: View {
                     // heavy -> extraBold : 800 숫자로 맞춰야 하는데 숫자를 fontWeight으로 주는 방법을 모르겠습니다.
                         .fontWeight(.heavy)
                 }
-                .padding(.top, 64)
+                .padding(.top, 30 + UIDevice.current.getSafeAreaTopValue)
```
Comment (unnnyong) [inline campair/campair/View/Editor/EditorDetailView.swift:51]: 나 ~~~~ 중에 코드를 읽더라도 바로 30의 의미를 알 수있게 30이 어떤 값인지 명명해서 사용하면 어떨까요 ?

```swift
// body 안
let topMargin: CGFloat = 30.0

Text("")
    ..padding(.top, topMargin + UIDevice.current.getSafeAreaTopValue)

Comment (rriver2) [reply campair/campair/View/Editor/EditorDetailView.swift:51]: When writing View UI code, do I have to put all the padding values as variables?
In the above code, 30 is the padding value of the painting card handed over by berry in sketch.
스크린샷 2022-06-14 오후 6 09 54
스크린샷 2022-06-14 오후 6 03 33

Comment (Gobans) [reply campair/campair/View/Editor/EditorDetailView.swift:51]: 🤔 I'm wondering it too, I'm afraid of if there are so many padding, so get complicated and hard to naming. Is there any good tip to handling style elements?

**Review by Gobans — APPROVED** [review]

File: campair/campair/View/Editor/EditorDetailView.swift (hunk 1 above)
Comment (unnnyong) [reply campair/campair/View/Editor/EditorDetailView.swift:51]: > Is there any good tip to handling style elemen

View의 extension method로 구현하는건 어떤가욥 ? (꿋꿋히 한글)

Comment (rriver2) [reply campair/campair/View/Editor/EditorDetailView.swift:51]: 땡큐영 ~

**Pull request merged**